src/components/shared/bf-rafter/BfRafter.vue

```diff
@@ -368,6 +373,10 @@ export default {
       return Object.keys(this.linkBack).length > 0
     },
 
+    isFileRecord: function() {
+      return this.$route.name == "file-record";
```

Comment:
can we change this to strict equals

Reply:
imo it makes sense to use strict equals (===) in all cases unless the condition is actually leveraging the type coersion we get using loose equals (==).